doc/api/packages.md

```diff
@@ -596,34 +603,30 @@ The `"import"`, `"require"`, `"node"`, `"node-addons"` and `"default"`
 conditions are defined and implemented in Node.js core,
 [as specified above](#conditional-exports).
 
-The `"node-addons"` condition can be used to provide an entry point which
-uses native C++ addons. However, this condition can be disabled via the
-[`--no-addons` flag][]. When using `"node-addons"`, it's recommended to treat
-`"default"` as an enhancement that provides a more universal entry point, e.g.
-using WebAssembly instead of a native addon.
-
 Other condition strings are unknown to Node.js and thus ignored by default.
 Runtimes or tools other than Node.js can use them at their discretion.
 
 These user conditions can be enabled in Node.js via the [`--conditions` flag][].
 
-The following condition definitions are currently endorsed by Node.js:
+Since user package conditions require clear definitions to ensure correct usage,
```

Comment:
I think this could be called out more clearly with a subheading.

Reply:
I've gone ahead and renamed the entire section to "community conditions definitions", along with further wording simplifications. Let me know if that works for you.